### DISCUSSION TITLE
Welcome to Teleport Discussions!

### DISCUSSION THREAD

**Discussion by klizhentas** [gravitational/teleport#5502] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by webvictim** [comment]
Hey everyone, I'm Gus - release engineer and general Devops guy on the Teleport team. Some of you may well have seen me around! 

Looking forward to the discussions :)

**Reply by hyvan29444** [reply]
Hello Gus, I am a young network engineer and I am trying to implement the Teleport solution in my organization and I would like to know if it is possible to connect Teleport to network equipment such as switches, routers and firewalls.
Thanks in advance

**Reply by webvictim** [reply]
Hey! it's possible to enable OpenSSH-enabled devices to trust Teleport-issued certificates in some cases - see our guide on OpenSSH hosts here: https://goteleport.com/docs/server-access/guides/openssh/#step-14-configure-sshd-to-trust-the-teleport-ca

In many cases network devices don't expose this configuration for you to modify, however. In that situation, my recommendation would to be deploy Teleport itself on a regular server so you can SSH into that, then run `ssh user@network-device-ip` on the command line there and enter your username/password as usual. That way the session will be recorded in full on the Teleport server and you also don't need to make changes to the network device configuration.

I hope this makes sense!
